app/main.py

```diff
@@ -1,3 +1,57 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open(os.path.join("app", "config.json"), "r") as f:
+        all_data = json.load(f)
+
+    customers_data = all_data["customers"]
+    shops_list = [
+        Shop(shop["name"], shop["products"], shop["location"])
+        for shop in all_data["shops"]
+    ]
+    fuel_price = all_data["FUEL_PRICE"]
+    for customer_data in customers_data:
+        current_customer = Customer(
+            customer_data["name"],
+            customer_data["money"],
+            customer_data["product_cart"],
+            customer_data["location"],
+            customer_data["car"],
+        )
```

Comment:
Same here. List comprehension and create customer like this `Customer(**customer_data)`
Using ** like this calles unpacking